scrapers/distgit.py

```diff
@@ -76,8 +76,11 @@ def update_neo4j(self, results):
                 'name': result['module']
             })[0]
             branch_name = result['ref'].rsplit('/', 1)[1]
-            branch = DistGitBranch.get_or_create(
-                {'name': branch_name}, relationships=repo.branches)[0]
+            branch = DistGitBranch.get_or_create({
+                'name': branch_name,
+                'repo_namespace': repo_info['namespace'],
```

Comment:
Do you think having a relationship might still be better? I feel this is duplicating data and can be achieved through a relationship

Reply:
My original plan was to do it through a relationship but that ended up being rather confusing (having multiple nodes with the same values but different relationships). @ralphbean and I decided this approach is preferable even if it is data duplication.